utils/health.py

```diff
@@ -0,0 +1,4 @@
+
+def update_health(name: str, value):
+    with open(f'/tmp/.health.{name.lower()}', 'w') as f:
+        f.write(str(value))
```

Comment:
I think this needs to delete the file if value is False?
value should also be typed as a bool

Reply:
alright